### PR TITLE
docs: new docs version, advanced plugin

### PR DESCRIPTION
**Pull request by banasa44**
## Description

Plugins section for advanced docs, percentage done by me:
- overview 50%
- Governance Plugins 100%
- PluginSetup Contract 50%
- Plugin Types 0%
- Plugin Versioning 0%
- Security 0%
- Framework 10% -> changed titles and not much more.

LEFT: fix all links!

Task: [OS-1382](https://aragonassociation.atlassian.net/browse/OS-1382)

[OS-1382]: https://aragonassociation.atlassian.net/browse/OS-1382?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/advanced/plugin/governance-plugins.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+- [A token-voting plugin](https://github.com/aragon/osx/tree/develop/packages/contracts/src/plugins/governance/majority-voting/token): Results are based on what the majority votes and the vote's weight is determined by how many tokens an account holds. Ex: Alice has 10 tokens, Bob 2, and Alice votes yes, the yes wins.
+- [Multisig plugin](https://github.com/aragon/osx/tree/develop/packages/contracts/src/plugins/governance/multisig): A determined set of addresses is able to approve. Once `x` amount of addresses approve (as determined by the plugin settings), then the proposal automatically succeeds.
+- [Admin plugin](https://github.com/aragon/osx/tree/develop/packages/contracts/src/plugins/governance/admin): One address can create and immediately execute proposals on the DAO (full control).
+- [Addresslist plugin](https://github.com/aragon/osx/tree/develop/packages/contracts/src/plugins/governance/majority-voting/addresslist): Majority-based voting, where list of addresses are able to vote in decision-making for the organization. Unlike a multisig, everybody here is expected to vote yes/no/abstain within a certain time frame.
```
Comment (Rekard0) [inline docs/advanced/plugin/governance-plugins.md:38]: these urls are not correct

File: docs/advanced/plugin/overview.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+sidebar_position: 0
+---
+
+## Aragon OSx Plugins
+
+As mentioned earlier, plugins built by Aragon and third-party developers can be added and removed from your DAO to adapt it to your needs.
+
+The management of these plugins is handled for you by the Aragon OSx protocol so that the process of
+
+- Releasing new plugins as well as
```
Comment (josemarinas) [inline docs/advanced/plugin/overview.md:13]: Mmm I'm not sure about this one is a little weird having the "as well as" in the same list no?

File: docs/advanced/plugin/overview.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+<p class="caption">
+  A schematic depiction of a plugin bundle consisting of a version tag, the plugin setup contract pointing to the plugin implementation contract, and a metadata URI.
+</p>
+
+</div>
+
+### What Does the Plugin Framework Consist Of?
+
+The Aragon OSx Plugin Framework consists of:
```
Comment (josemarinas) [inline docs/advanced/plugin/overview.md:53]: Same as the previous section

File: docs/advanced/plugin/overview.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  - listing all the plugins managed through the Aragon OSx protocol
+
+- The `PluginRepo` contract
+
+  - versions the releases of a Plugin
+
+<div class="center-column">
+
+![](/optimized-svg/framework/aragon-os-plugin-framework.drawio.svg)
```
Comment (josemarinas) [inline docs/advanced/plugin/overview.md:70]: I would try to use the same colors and style as in the rest of schemas

File: docs/advanced/plugin/overview.md (hunk 3 above)
Comment (banasa44) [reply docs/advanced/plugin/overview.md:53]: but they would have later a fully dedicated page for themselves (same above)
I see this page as an overview of what you're going to see in the Advanced-Plugin section.
Maybe I could add a page for Plguin (previous to governance plugins page)
And another for metadata URI...? or add a section for it in Plugin?

File: docs/advanced/plugin/governance-plugins.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+While a DAO can technically function without a governance plugin by granting the `EXECUTE_PERMISSION_ID` to an EOA, this practice is generally discouraged as it defeats the purpose of decentralized governance.
+
+### Execute Permission
+
+The `EXECUTE_PERMISSION_ID` is one of the most critical permissions within your DAO. It allows the entity holding this permission, whether an EOA or a smart contract, to call the DAO's `execute` function. This function is essential for carrying out decisions and actions within the DAO.
+
+### Governance Plugin
+
+Governance plugins are characterized by the **ability to execute actions in the DAO** they have been installed on. Accordingly, the `EXECUTE_PERMISSION_ID` is granted to the governance plugin contract upon installation in the DAO. If using Aragon OSx protocol (DAOFactory `createDao` function) at least one plugin installation is required when deploying the DAO.
```
Comment (josemarinas) [inline docs/advanced/plugin/governance-plugins.md:17]: I would say something in the lines

Goverannce plugins should be able to execute actions on behalf of the dao so the EXECUTION_PERMISSION must be granted to this type of plugins via the PluginSetup. 

The line about the createDao I'm not sure it fits here

**Review by clauBv23 — APPROVED** [review]

**Pull request merged**